### PR TITLE
Deploy information and general updates to README.md 

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
Several features present in https://github.com/department-of-veterans-affairs/vets-api/blob/master/README.md are not setup.

Deploy information was missing information about post deploy deployment actions

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -120,17 +121,39 @@ To work on your code submission, follow [GitHub Flow](https://guides.github.com/
 
 1. Branch or Fork
 1. Commit changes
-1. Submit Pull Request
+1. Create Draft Request
```
[the remark this answers is not in the document]
Comment (zurbergram) [reply README.md]: changed

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ Note that queries are PostgreSQL-specific.
 - `bundle exec guard` - Runs the guard test server that reruns your tests after files are saved. Useful for TDD!
 - `bundle exec rake security` - Run the suite of security scanners on the codebase.
 - `bundle exec rake ci` - Run all build steps performed in Travis CI.
```
Comment (jholton) [inline README.md]: Noticed this.  Can you remove "Travis" since we do CI ourselves (jenkins) now?

Comment (zurbergram) [reply README.md]: updated description

**Review by jholton — APPROVED** [review]

**Pull request merged**